app/javascript/src/pledge_calculator.js

```diff
@@ -0,0 +1,68 @@
+var updateBalance = function() {
+    if ($("#patient_procedure_cost").val()) {
+        $(".outstanding-balance-ctn").removeClass('hidden');
+        return $("#outstanding-balance").text('$' + calculateRemainder());
+    } else {
+        return $(".outstanding-balance-ctn").addClass('hidden');
+    }
+};
+
+var calculateRemainder = function() {
+    var contributions, total;
```

Comment:
Here's another place where we can use javascript's `let` and `const` to tell other programmers that something's going to stay the same after we declare it, or that we're going to manipulate it. In this case, we're setting `total`, setting `contributions` by doing some fancy math, and then returning the difference. So we aren't really changing these once we're setting them -- what do you think about removing this line, and then doing `total = valueToNumber($("#patient_procedure_cost").val());` and `const contributions = [BUNCH OF FANCY MATH]`? In your opinion, does that make sense?